modularbuild/abstracts.yaml

```diff
@@ -0,0 +1,35 @@
+#Base components for setting up a deploy configuration. 
+
+.migrate:
+  image: briangweber/docker-node:carbon
+  script:
+    - cp -r $CI_PROJECT_DIR/sql/* /flyway/sql/
+    - sh /flyway/flyway -url=jdbc:postgresql://$POSTGRES_SERVER:5432/$POSTGRES_DB -schemas=public -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD -placeholders.ENVIRONMENT=$CI_ENVIRONMENT_NAME migrate
+
+.deploy:
+  script:
+    - grunt deploy --environment=$CI_ENVIRONMENT_NAME --packageVersion=$VERSION --packageName=$CI_PROJECT_NAME
```

Comment:
This can't be used without the node before script, so it effectively cannot be shared. See other resolved comments about this.